README.md

```diff
@@ -1,2 +1,19 @@
-# nodejs-aws-be
-AWS BE services
+## TASK 3
+
+###FE App with dynamic products from BE:
+ https://d2q9oyzdq8h9iv.cloudfront.net/
+ 
+ #### FE github link:
+ https://github.com/Panibrat/nodejs-aws-fe
```

Comment:
//import productList from "./productList.json";  NOT deleted from FE .. Looks like it's not needed anymore

Reply:
//setProducts(productList);  Comment is redudndant